### ISSUE TITLE
Minimum autobahn appears to be greater than 0.13.0

### ISSUE THREAD

**Issue by sachinrekhi** [django/daphne#101]
I just upgraded from daphne 0.12.0 to 1.1.0. When I subsequently started the dev server, I got the following error:

```
Starting Channels development server at http://0.0.0.0:8000/
Channel layer default (asgi_redis.core.RedisChannelLayer)
Quit the server with CONTROL-C.
2017-03-30 19:02:00,441 - INFO - worker - Listening on channels http.request, websocket.connect, websocket.disconnect, websocket.receive
2017-03-30 19:02:00,445 - INFO - worker - Listening on channels http.request, websocket.connect, websocket.disconnect, websocket.receive
2017-03-30 19:02:00,446 - INFO - worker - Listening on channels http.request, websocket.connect, websocket.disconnect, websocket.receive
2017-03-30 19:02:00,446 - INFO - worker - Listening on channels http.request, websocket.connect, websocket.disconnect, websocket.receive
Unhandled exception in thread started by <function wrapper at 0x108c7c140>
Traceback (most recent call last):
  File "/Users/sachin/Documents/Code/virtualenvs/notejoy/lib/python2.7/site-packages/django/utils/autoreload.py", line 226, in wrapper
    fn(*args, **kwargs)
  File "/Users/sachin/Documents/Code/virtualenvs/notejoy/lib/python2.7/site-packages/channels/management/commands/runserver.py", line 90, in inner_run
    root_path=getattr(settings, 'FORCE_SCRIPT_NAME', '') or '',
  File "/Users/sachin/Documents/Code/virtualenvs/notejoy/lib/python2.7/site-packages/daphne/server.py", line 80, in run
    proxy_forwarded_port_header=self.proxy_forwarded_port_header
  File "/Users/sachin/Documents/Code/virtualenvs/notejoy/lib/python2.7/site-packages/daphne/http_protocol.py", line 314, in __init__
    allowNullOrigin=True,
TypeError: setProtocolOptions() got an unexpected keyword argument 'allowNullOrigin'
```

It appeared to be an interface difference with the interface Daphne was expecting with autobahn. I then upgraded autobahn from what I had installed (0.13) to the latest (0.18.1) and this error went away.

It thus appears that the minimum required version for autobahn is now higher than the 0.12 reflected in setup.py

**Comment by pamungkaski** [comment]
already make a pull request for this issue. But fail in the travis CI i don't know why because i only fork then change the setup.py and push again without touching any other file #102

**Comment by andrewgodwin** [comment]
Fixed in 1.2.0.
